src/main/java/kitchenpos/dto/order/OrderTableResponse.java

```diff
@@ -0,0 +1,51 @@
+package kitchenpos.dto.order;
+
+import kitchenpos.domain.order.OrderTable;
+
+import java.util.Objects;
+
+public class OrderTableResponse {
+    private Long id;
+    private Long tableGroupId;
+    private int numberOfGuests;
+    private boolean empty;
+
+    private OrderTableResponse() {
+    }
+
+    public OrderTableResponse(Long id) {
+        this.id = id;
+    }
+
+    public OrderTableResponse(Long id, Long tableGroupId, int numberOfGuests, boolean empty) {
+        this.id = id;
+        this.tableGroupId = tableGroupId;
+        this.numberOfGuests = numberOfGuests;
+        this.empty = empty;
+    }
+
+    public static OrderTableResponse of(OrderTable orderTable) {
+        if (Objects.isNull(orderTable.getTableGroup())) {
+            return new OrderTableResponse(orderTable.getId(), null,
+                    orderTable.getNumberOfGuestsCount(), orderTable.isEmptyTable());
+        }
+        return new OrderTableResponse(orderTable.getId(), orderTable.getTableGroup().getId(),
+                orderTable.getNumberOfGuestsCount(), orderTable.isEmptyTable());
```

Comment:
```suggestion
        return new OrderTableResponse(orderTable.getId(), orderTable.getTableGroupId(),
                orderTable.getNumberOfGuestsCount(), orderTable.isEmptyTable());
```

와 같이 사용한다면 조금 더 깔끔한 코드로 작성할 수 있겠네요.
이러한 스타일은 취향차이라 참고만 해주셔도 괜찮습니다!

Reply:
다른 response에 대해서는 작성했는데 여기에는 빼먹었네요!
감사합니다 :)